pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -663,13 +627,19 @@ class Build {
 
         context.node('worker') {
             context.stage("installer") {
-                // Ensure master context workspace is clean of any previous archives
-                context.sh "rm -rf workspace/target/* || true"
```

Comment:
These workspace clear need to remain. the reason they have to be target/* is because a previous installer invocation on the "worker" node will potentially still have old artifacts there.
For example, even though say we are doing a Mac Installer, there maybe a previous Win Installer job .zip file there...

Reply:
should not https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/build/common/openjdk_build_pipeline.groovy#L1463 be triggered for the cleanup as post step by previous run?

Reply:
we should be using context.copyArtifacts() ....